radio/src/CMakeLists.txt

```diff
@@ -442,6 +442,7 @@ if(ARCH STREQUAL ARM)
     ppm_arm.cpp
     pxx_arm.cpp
     dsm2_arm.cpp
+    multi_arm.cpp
```

Comment:
The Cmake file is just guessing at the moment. I am trying to get cmake running (include could not find load file:    Bitmaps)

When I get it building I will submet a fixed Cmake file